docs/content/docs/data-structures/counter.md

```diff
@@ -1,5 +1,141 @@
+A new `SharedCounter` value will be initialized with its value set to `0`.
+If you wish to initialize the counter to a different value, you may [modify the value](#incrementing--decrementing-the-value) before attaching the Container, or before inserting it into an existing shared object like a `SharedMap`.
+
+### Incrementing / decrementing the value
+
+Once you have created your `SharedCounter`, making changes to the value is as simple as calling its [increment]({{< relref "isharedcounter.md#increment-MethodSignature" >}}) method.
+This method accepts a positive or negative *integer* delta to be applied to the shared value.
+
+- Note:
+
+```javascript
+sharedMap.increment(3); // Adds 3 to the current value
+sharedMap.increment(-5); // Subtracts 5 from the current value
+```
+
+### [incremented]({{< relref "isharedcounterevents.md#_call_-CallSignature" >}}) event
+
+The `incremented` event is sent when a client in the collaborative session changes the counter value via `increment`.
+
+> Signature: `(event: "incremented", listener: (incrementAmount: number, newValue: number) => void)`
+
+By registering with this event, you can receive and apply the necessary deltas coming from other collaborators.
+Consider the following code example for configuring a Counter widget:
+
+```javascript
+const sharedCounter = container.initialObjects.sharedCounter;
+let counterValue = sharedCounter.value;
+
+const incrementButton = document.createElement('button');
+button.textContent = "Increment";
+const decrementButton = document.createElement('button');
+button.textContent = "Decrement";
+
+// Increment / decrement shared counter value when the corresponding button is clicked
+incrementButton.addEventListener('click', () => sharedCounter.increment(1));
+decrementButton.addEventListener('click', () => sharedCounter.increment(-1));
+
+const counterValueLabel = document.createElement('label');
+counterValueLabel.textContent = `${counterValue}`;
+
+// This function will be called each time the shared counter value is incremented
+// (including increments from this client).
+// Update the local counter value and the corresponding label being displayed in the widget.
+const updateCounterValueLabel = (delta) => {
+    counterValue += delta;
+    counterValueLabel.textContent = `${counterValue}`;
+};
+
+// Register to be notified when the counter is incremented
+sharedCounter.on("incremented", updateCounterValueLabel);
+```
+
+In the code above, whenever a user presses either the `Increment` or `Decrement` button, the shared `sharedCounter.increment` is called with +/- 1.
+This causes the `incremented` event to be sent to all of the clients who have this container open.
+
+Since `updateCounterValueLabel` is registered for all `incremented` events, the view will always refresh with the appropriate updated value any time a collaborator increments or decrements the counter value.
+
```

Comment:
I recommend that here we have a paragraph that returns to what was said at the top of the article about "the pitfalls of DDSes with simpler merge strategies, in which one user's edit may clobber another's". What exactly could go wrong if the SharedCounter wasn't architected in the right way? How is that problem avoided?

Reply:
I added a sub-heading under Introduction to explain this. Let me know what you think :)